tools/compute_power_budget.py

```
@@ -0,0 +1,349 @@
+            )
+            print(
+                "     Power caps will prevent all GPUs from sustaining TDP "
+                "simultaneously — this is expected."
+            )
+            print()
+        print("=" * 60)
+
+        print()
+        print("  Additional planner settings to review:")
+        print(f"  power_agent_safe_default_watts: {math.ceil(self.gpu_tdp_w * 0.70)}")
+        print(f"    (= 70% of TDP; adjust for your model's idle power profile)")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)